src/Microsoft.AspNet.Mvc.DataAnnotations/DataAnnotationsClientModelValidatorProvider.cs

```diff
@@ -46,14 +68,14 @@ public void GetValidators(ClientValidatorProviderContext context)
                 DataAnnotationsClientModelValidationFactory factory;
                 if (_attributeFactories.TryGetValue(attribute.GetType(), out factory))
                 {
-                    context.Validators.Add(factory(attribute));
+                    context.Validators.Add(factory(attribute, stringLocalizer));
```

Comment:
Din't feel it adds a lot of value here. The adapters already have `IStringLocalizer` as constructor argument. So passing stringLocalizer here. If resource is not configured, `stringLocalizer` would be null. Let me know if you disagree